app/views/user_batches/form.html.erb

```diff
@@ -1,7 +1,35 @@
 <%= elmo_form_for(@user_batch, html: {multipart: true}) do |f| %>
   <p class="instructions"><%= formatted_instructions %></p>
-  <%= f.field(:file, type: :file, accept: ".xlsx,.csv", required: true) %>
+
+  <div id="file-uploader">
+    <%= hidden_field_tag(:temp_file_path, "", id: "temp_file_path") %>
+    <%= hidden_field_tag(:original_filename, "", id: "original_filename") %>
+
+    <div id="dropzone-file-uploader" class="dropzone">
+      <div class="dz-message">
+        <span class="message"><%= t("common.file_drop_message") %>&nbsp;</span>
+      </div>
+    </div>
+    <div class="dz-error-msg"></div>
+  </div>
   <div class="submit-buttons">
     <%= f.submit(t("common.import"), class: "btn btn-primary") %>
+    <%= render("file_upload/upload_progress") %>
```

Comment:
Perhaps could be more specific and call it `upload_in_progress_button`? The actual upload progress is handled by dropzone...